game/state/gameevent.cpp

```diff
@@ -370,11 +370,11 @@ GameSomethingDiedEvent::GameSomethingDiedEvent(GameEventType type, UString name,
 		case GameEventType::BaseDestroyed:
 			if (actor.length() > 0)
 			{
-				messageInner = tr("X-COM base destroyed by hostile forces.");
+				messageInner = tr("X-COM " + name + " destroyed by hostile forces.");
```

Comment:
This doesn't work well with translations - something like "format(tr("X-COM %s destroyed by hostile forces"), name)" should allow the string to be translated.

tr() should only take static strings, as that can then be extracted for translation.

Reply:
Oh right didn't think about that.